lua/nvim-tree/api.lua

```diff
@@ -160,18 +159,12 @@ end)
 
 Api.tree.change_root_to_node = wrap_node(function(node)
   if node.name == ".." or node:is(RootNode) then
-    actions.root.change_dir.fn("..")
-    return
-  end
-
-  if node:is(FileNode) and node.parent ~= nil then
-    actions.root.change_dir.fn(node.parent:last_group_node().absolute_path)
```

Comment:
We cannot remove this case. We need to be able to `<C-]>` on a file node as per help:

"Change the tree's root to a folder node or the parent of a file node."